modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -489,6 +637,37 @@ private void _loadObjectDefinition(Map<String, Serializable> parameters)
 		throw new NotFoundException("Missing parameter \"objectDefinitionId\"");
 	}
 
+	private Page<Permission> _toPermissionPage(
```

Comment:
I don't get this method. Is not the same as [this one](https://github.com/liferay-headless/liferay-portal/blob/aa7f9d5bc1c18100b107f90124fdc287756aa9b2/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/BaseObjectEntryResourceImpl.java#L1140-L1170) that it is in the Base?
Or I'm missing something?? I'm doing a quick review so maybe I'm missing something 😁

Reply:
You are right @LuismiBarcos , I didn't figure out that toPermissionPage is a protected method :sweat_smile:. I'll delete it